frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -699,7 +709,16 @@ public HashMap<String, String> getBackendConfiguration() {
         HashMap<String, String> config = new HashMap<>();
         // Append properties used by backend worker here
         config.put("TS_DECODE_INPUT_REQUEST", prop.getProperty(TS_DECODE_INPUT_REQUEST, "true"));
+        
         config.put("TS_IPEX_ENABLE", prop.getProperty(TS_IPEX_ENABLE, "false"));
+        config.put("TS_IPEX_DTYPE", prop.getProperty(TS_IPEX_DTYPE, "float32"));
+        config.put("TS_IPEX_CHANNEL_LAST", prop.getProperty(TS_IPEX_CHANNEL_LAST, "true"));
+        config.put("TS_IPEX_TORCHSCRIPT", prop.getProperty(TS_IPEX_TORCHSCRIPT, "true"));
+        config.put("TS_IPEX_INPUT_TENSOR_SHAPES", prop.getProperty(TS_IPEX_INPUT_TENSOR_SHAPES, null));
+        config.put("TS_IPEX_INPUT_TENSOR_DTYPE", prop.getProperty(TS_IPEX_INPUT_TENSOR_DTYPE, null));
+        config.put("TS_IPEX_QSCHEME", prop.getProperty(TS_IPEX_QSCHEME, "per_tensor_affine"));
+        config.put("TS_ONEDNN_GRAPH_FUSION_ENABLE", prop.getProperty(TS_ONEDNN_GRAPH_FUSION_ENABLE, "false"));
+        
```

Comment:
Same here, they should be each model's config if each model can have different setting.

Reply:
And also, as I commented in [issue1631](https://github.com/pytorch/serve/issues/1631#issuecomment-1129443165), it is not safe by setting environment. We need a secure solution.